src/Essentials/src/MediaGallery/MediaGallery.uwp.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Storage;
+using Windows.Media.Capture;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+
+namespace Microsoft.Maui.Media
+{
+	partial class MediaGalleryImplementation
+	{
+		public bool IsSupported => true;
+
+		public bool CheckCaptureSupport(MediaFileType type) => true;
+
+		public async Task<IEnumerable<MediaFileResult>> PlatformCaptureAsync(MediaFileType type, CancellationToken token = default)
+		{
+			var captureUi = new CameraCaptureUI();
+			var photo = type == MediaFileType.Image;
+
+			if (photo)
+				captureUi.PhotoSettings.Format = CameraCaptureUIPhotoFormat.Jpeg;
+			else
+				captureUi.VideoSettings.Format = CameraCaptureUIVideoFormat.Mp4;
+
+			var file = await captureUi.CaptureFileAsync(photo ? CameraCaptureUIMode.Photo : CameraCaptureUIMode.Video);
+
+			if (file != null)
+				return new[] { new MediaFileResult(file) };
+
+			return null;
```

Comment:
Either do not return null or change the signature. Array.Empty would work here as with the previous comment.